timescale-forge/vpc-peering.md

```diff
@@ -0,0 +1,141 @@
+
+### Network Route & Security
+Once you have accepted the peering connection from the last step, the two VPCs will now
+be peered; however, in order to use this peering connection, you will need to update your
+VPC's route table to include the CIDR block of your peered Forge VPC, and lastly you will
+need to update your VPC's security groups.
+
+#### Route Table
+Within the AWS console, navigate to the
+[Route Tables](https://console.aws.amazon.com/vpc/home#RouteTables:sort=routeTableId)
+dashboard. Select the route table corresponding to your VPC. From the detail menu, select
+the `Routes` tab and click the `Edit routes` button.
+
+<img class="main-content__illustration" src="aws-route-table-routes.png" alt="The AWS Route Tables dashboard with Routes tab expanded"/>
+
+From this view, click `Add route`. In the `Destination` column of the new row,
+enter the CIDR block of the Forge VPC for which the peering connection was
+configured in the previous few steps.
+
+Then in `Target` column, enter the peering connection ID (starting with `pcx-`)
+noted in the previous step where we created the peering connection.
+
+No other configuration is needed here, so you may now click `Save routes`. This
+configuration allows network traffic to flow from your VPC, across the peering
+connection, and over to the Forge VPC where your Timescale services reside.
+
+<img class="main-content__illustration" src="aws-edit-routes.png" alt="Adding a new route table entry for our peering connection"/>
+
+#### Security Groups
+Within the AWS console, navigate to the
+[Security Groups](https://console.aws.amazon.com/vpc/home#securityGroups:)
+dashboard. From this view, click `Create security group` to create a new security group.
+
+<highlight type="tip">
+If needed, another security group which already exists in your VPC may be used;
+however, for simplicity we will assume the creation of a new security group.
+</highlight>
+
+<img class="main-content__illustration" src="aws-create-security-group.png" alt="The AWS Security Groups dashboard"/>
+
+From the Create security group view, enter a name for your security group. Add whatever
+content you would like to the description field. For the VPC field, select the VPC
+which has been peered with your Forge VPC.
+
+No inbound rules are required, so leave the inbound rules section empty.
+
+In the outbound rules section, select `Custom TCP` for the rule type. The protocol
+should remain as TCP. The port range should be `5432`, which is the port which will
+be used to connect to your Timescale Forge services. The Destination should be set
+to `Custom` and the value should be the CIDR block of your Forge VPC.
+
+Finally, click `Create security group`. With this step, you will now be able to
+connect to any of your Timescale Forge services attached to your peered VPC.
```

Comment:
Do we want a quick section that also describes how to move services between VPCs, or from VPC -> Public -> VPC?

Also, any section about the status codes of the VPC?

Reply:
I'm writing the docs for service migrations right now. I'll make sure to ping you for review once those are done.

Reply:
> Also, any section about the status codes of the VPC?

What did you have in mind for this? Just a description of the various states which are presented in the Forge UI?